jberet-core/src/main/java/org/jberet/repository/JdbcRepository.java

```diff
@@ -146,11 +146,23 @@ public JdbcRepository(final Properties configProperties) {
             this.dbUrl = dbUrl;
             final String dbUser = configProperties.getProperty(DB_USER_KEY);
             if (dbUser != null) {
-                dbProperties.setProperty("user", dbUser.trim());
+                if (dbUser.startsWith("${")){
+                    final String dbPasswordFromEnvVar = parseValuefromEnvVariables(dbUser);
+                    dbProperties.setProperty("user", dbPasswordFromEnvVar);
+                }
+                else {
+                    dbProperties.setProperty("user", dbUser.trim());
+                }
             }
             final String dbPassword = configProperties.getProperty(DB_PASSWORD_KEY);
             if (dbPassword != null) {
-                dbProperties.setProperty("password", dbPassword.trim());
+                if (dbPassword.startsWith("${")){
+                    final String dbPasswordFromEnvVar = parseValuefromEnvVariables(dbPassword);
+                    dbProperties.setProperty("password", dbPasswordFromEnvVar);
+                }
+                else {
+                    dbProperties.setProperty("password", dbPassword.trim());
+                }
```

Comment:
I think the check should be a bit stricter whether it's a clear text value or expression. For example, `${xxx` can be a perfectly valid clear-text password, not an expression.

We could check if it follows the pattern of `${.*}`, meaning it should start with `${` and end with `}` with some chars in between.